src/pages/home/report/ReportFooter.js

```diff
@@ -54,14 +59,15 @@ const defaultProps = {
     lastReportAction: null,
     isEmptyChat: true,
     shouldShowComposeInput: false,
+    session: {},
 };
 
 function ReportFooter(props) {
     const styles = useThemeStyles();
     const {isOffline} = useNetwork();
     const chatFooterStyles = {...styles.chatFooter, minHeight: !isOffline ? CONST.CHAT_FOOTER_MIN_HEIGHT : 0};
     const isArchivedRoom = ReportUtils.isArchivedRoom(props.report);
-    const isAnonymousUser = Session.isAnonymousUser();
+    const isAnonymousUser = props.session.authTokenType === 'anonymousAccount';
```

Comment:
We can move string `anonymousAccount` to const file as it is used in multiple places.

Reply:
Done